controllers/keywordController.js

```diff
@@ -0,0 +1,49 @@
+const keywordModel = require("../models/keywordModel");
+const groupModel = require("../models/groupModel");
+
+const isValid = (data) => {
+  return data !== null && data !== undefined;
+};
+
+const create = async (req, res) => {
+  if (!isValid(req.body.groupName) || !isValid(req.body.keyword) || !isValid(req.body.ownerId)) {
+    res.status(400).send({ message: "[Error: keywordCreate] Content can not be empty!" });
+  }
+
+  const isDuplicatedKeyword =
+    keywordModel.find({ groupId: req.body.groupId, keyword: req.body.keyword }) === null;
+
+  if (isDuplicatedKeyword) {
+    res.status(400).send({ message: "[Error: keywordCreate] Content can not be duplicated!" });
+  }
+
+  const { groupId, groupName, keyword, ownerId } = req.body;
+
+  try {
+    if (groupId === null) {
+      const resultGroupCreated = await groupModel.create({
+        name: groupName,
+        ownerId,
+        keywordList: [keyword],
+      });
+      await keywordModel.create({
+        keyword,
+        ownerId: ownerId,
```

Comment:
```suggestion
        ownerId,
```
key와 value 이름이 ownerId로 동일하기 때문에, 통일성을 위해 상단의 keyword처럼 해당 부분도 shorthand property를 사용하면 좋을 것 같습니다.

Reply:
앗 ownerId를 못 챙겼네요! 수정하도록 하겠습니다 :)